tests/unit/test_column_noise.py

```diff
@@ -399,34 +402,28 @@ def test_generate_copy_from_household_member(
 def test_swap_months_and_days(dataset: Dataset, fuzzy_checker: FuzzyChecker) -> None:
     for col in ["event_date", "date_of_birth"]:
         data = dataset.data[col].copy()
-        if col == "event_date":
-            config: LayeredConfigTree = get_configuration()[DATASET_SCHEMAS.ssa.name][
-                Keys.COLUMN_NOISE
-            ][col][NOISE_TYPES.swap_month_and_day.name]
-            config.update(
-                {
-                    DATASET_SCHEMAS.ssa.name: {
-                        Keys.COLUMN_NOISE: {
-                            col: {
-                                NOISE_TYPES.swap_month_and_day.name: {
-                                    Keys.CELL_PROBABILITY: 0.25,
-                                },
+        config: NoiseConfiguration = get_configuration()
+        config._update(
+            {
+                DATASET_SCHEMAS.census.name: {
```

Comment:
How are you ablew to get rid of the if/else logic here? I don't recall the details of why that is (was?) needed, but evidently "event_date" is only an SSA dataset col.

Reply:
You caught something here - this test should have been using different probabilities for event date and date of birth to reproduce the old test behavior, but it was always changing it to 0.25. I've made a fix in my most recent commit.